software/contrib/consequencer.md

```diff
@@ -86,20 +97,38 @@ Example usage for these CV outputs, (this will depend on the inputs your drum mo
 ## Selecting analogue input modes
 
 Consequencer can perform 3 different actions when a control voltage input is received at the analogue input.
-The current running mode is shown on the bottom right of the screen (e.g. M1, M2, M3)
+The current running mode is shown on the bottom right of the screen (e.g. Mr, Mp, Mc)
 Cycle through the modes by long-pressing and releasing button 2. The following modes are available:
 
-- Mode 1: Control voltage adjusts randomness of the gate patterns sent to outputs 1 - 3
-- Mode 2: Control voltage selects the gate pattern
-- Mode 3: Control voltage selects the stepped CV pattern
+- Mode 1 (Mr) : Control voltage adjusts randomness of the gate patterns sent to outputs 1 - 3
+- Mode 2 (Mp) : Control voltage selects the gate pattern
+- Mode 3 (Mc) : Control voltage selects the stepped CV pattern
 
 ## Controlling a pattern using CV
 
-1. Select analogue mode 2.
+1. Select analogInputMode 2 (Mr) using button 2.
 2. Send a control voltage into the analogue input
 
 A fixed voltage will select a single pattern and varying voltage (e.g. an envelope or LFO) will smoothly cycle through patterns.
 
+# Grids Patterns
+
+The 25 different patterns from the Mutable Instruments Grids module were exported from the Grids code and added to the Consequencer.
+
+Access the Grids patterns by holding button 2 for 3-5 seconds and releasing.
+Grids mode is enabled a period '.' before the pattern number on the bottom right of the screen.
```

Comment:
Spelling/Rewording - I think this is supposed to be "When Grids mode is enabled, a period '.' is visible before the pattern number on the bottom right of the screen"